android/app/src/main/kotlin/com/chicagoroboto/ext/ViewExt.kt

```diff
@@ -1,6 +1,17 @@
 package com.chicagoroboto.ext
 
 import android.app.Activity
+import android.content.ContextWrapper
 import android.view.View
 
-fun View.getActivity(): Activity? = if (context is Activity) context as Activity? else null
+fun View.getActivity(): Activity? {
+  var context = context
+  while (context !is Activity) {
```

Comment:
I'm not familiar with this, does the `!is` protect against null, so this will exit if `context == null`?